client/js/socket.js

```diff
@@ -11,6 +11,10 @@ const socket = io({
 	reconnection: !$(document.body).hasClass("public"),
 });
 
+$("#loading-page-message, #connection-error").on("click", function() {
```

Comment:
Remove `#loading-page-message` here, we don't want it clickable.

Reply:
done